scripts/test/release.sh

```diff
@@ -94,5 +94,6 @@ test_release '100.0.0-rc0' 'branch' 'branch: release-100.0' 'pre-release: true'
 
 # Test with an existing known branch
 # Only releases or release candidates are expected to work as we're not releasing milestones on stable branches
-test_release '0.9.100' 'shipyard' 'branch: release-0.9'
-test_release '0.9.100-rc1' 'shipyard' 'branch: release-0.9' 'pre-release: true'
+branch=$(git rev-parse --symbolic --remotes='submariner/release-*' | grep -o -m 1 'release-[0-9]*\.[0-9]*')
```

Comment:
If you want the latest stable branch:

```suggestion
branch=$(git rev-parse --symbolic --remotes='submariner/release-*' | sort -rV | grep -o -m 1 'release-[0-9]*\.[0-9]*')
```